soracom-inventory-agent-for-java-core/src/main/java/io/soracom/inventory/agent/core/initialize/InventoryAgentInitializer.java

```diff
@@ -322,10 +322,8 @@ private void initServer(ObjectsInitializer initializer, Credentials credentials)
 		if (objectInstanceMap.containsKey(LwM2mId.SERVER)) {
 			return;
 		}
-		if (credentials != null) {
-			serverObject = InventoryAgentHelper.getServerInfo(credentials);
-			initializer.setInstancesForObject(LwM2mId.SERVER, serverObject);
-		}
```

Comment:
what intention is this null check removing?

Reply:
I updated InventoryAgentHelper.getServerInfo() to return Server object even credentials is null, so I needed to remove the null check.
https://github.com/soracom/soracom-inventory-agent-for-java/pull/14/files/706928bb77e5ab315e28df642835f58343c3a3e2#diff-2d9f8e0b00b3293c40ed7cb2973ee656e4d3f2bc288a6b231fac79059d085c30R65